_episodes/01-intro.md

```diff
@@ -181,21 +181,24 @@ weight in kilograms is now: 65.0
 
 > ## Check Your Understanding
 >
-> What values do the variables `mass` and `age` have after each statement in the following program?
-> Test your answers by executing the commands.
+> What values do the variables `mass` and `age` have after each of the following statements? 
+> Test your answer by executing the lines.
 >
 > ~~~
 > mass = 47.5
 > age = 122
 > mass = mass * 2.0
 > age = age - 20
-> print(mass, age)
+> 
 > ~~~
 > {: .language-python}
 >
 > > ## Solution
 > > ~~~
-> > 95.0 102
+> > `mass` holds a value of 47.5, `age` does not exist
+> > `mass` still holds a value of 47.5, `age` holds a value of 122
+> > `mass` now has a value of 95, `age`'s value is still 122
+> > `mass` still has a value of 95, `age` now holds 102
```

Comment:
I like where you have taken this @ncf0003. :)  I just noticed that we should perhaps use 95.0 as a float instead of the int 95 to prevent possible confusion. Thank you for your work on this, Nicolas!

Reply:
Very good call @ldko! I updated it and I'll see you around!

Reply:
Thank you, both!